src/opus/opus.cc

```diff
@@ -181,6 +181,15 @@ static bool update_replay_gain(OggOpusFile * opus_file,
 
     const char * album_gain = opus_tags_query(tags, "R128_ALBUM_GAIN", 0);
     const char * track_gain = opus_tags_query(tags, "R128_TRACK_GAIN", 0);
+    bool got_r128 = true;
+
+    /* try to read RG tags if R128 didn't work*/
+    if (!album_gain && !track_gain)
+    {
+        got_r128 = false;
+        album_gain = opus_tags_query(tags, "REPLAYGAIN_ALBUM_GAIN", 0);
+        track_gain = opus_tags_query(tags, "REPLAYGAIN_TRACK_GAIN", 0);        
+    }
```

Comment:
I would prefer this approach:

```c++
bool has_r128_tags = (album_gain && track_gain);

/* try to read RG tags if R128 didn't work */
if (!has_r128_tags)
{
    album_gain = opus_tags_query(tags, "REPLAYGAIN_ALBUM_GAIN", 0);
    track_gain = opus_tags_query(tags, "REPLAYGAIN_TRACK_GAIN", 0);        
}
```

Reply:
That makes sense. I was unsure of a clear way to have it.